tests/criticism/test_criticism.py

```diff
@@ -27,20 +28,20 @@ def get_ppc_with_samples(adata: AnnData, n_samples: int = 2):
     model2.train(1)
 
     models_dict = {"model1": model1, "model2": model2}
-    ppc = PPC(adata, models_dict, n_samples=n_samples)
+    ppc = PPC(adata, models_dict, n_samples=n_samples, indices=indices)
     return ppc, models_dict
 
 
 def test_ppc_init():
     adata = synthetic_iid()
-    ppc, models_dict = get_ppc_with_samples(adata, n_samples=42)
+    ppc, models_dict = get_ppc_with_samples(adata, n_samples=42, indices=np.arange(100))
```

Comment:
We should also keep a call to `get_ppc_with_samples` without passing in `indices`.